cmd/podman/top.go

```diff
@@ -27,13 +27,13 @@ var (
 	topCommand     cliconfig.TopValues
 	topDescription = fmt.Sprintf(`Similar to system "top" command.
 
-  Specify format descriptors to alter the output.
+  Specify format descriptors to alter the output. Alternatively, you can specify flags and options for ps(1).
```

Comment:
I wouldn't document/mention this... I view this more as a compatibility thing than something we actually want to advertise as a feature